src/session_pool/session_pool.js

```diff
@@ -172,12 +172,17 @@ export class SessionPool extends EventEmitter {
      * If there is space for new session, it creates and return new session.
      * If the session pool is full, it picks a session from the pool,
      * If the picked session is usable it is returned, otherwise it creates and returns a new one.
-     *
+     * @param {String} sessionId
```

Comment:
The param should be optional.

Reply:
Yes, wasn't sure how to write it properly, then forgot about it